### ISSUE TITLE
Recommendation to use a XML parser

### ISSUE THREAD

**Issue by 304NotModified** [JTOne123/GitHubMassUpdater#4]
HI

I like your tool and checked in short your source code. 

I see you're using a text search for replacing stuff in the XML.

I would recommend to use a XML Parser for that, for example XElement.Parse - it's less error prone.

**Comment by JTOne123** [comment]
yes, good idea, what about formatting? XML Parser that you suggest is using the original formatting?
